tests/unit/runtime/zero/test_offload_states.py

```diff
@@ -124,5 +124,8 @@ def test_offload_states(self, included_state, pin_memory, non_blocking):
         with deepspeed.zero.Init(config_dict_or_path=config_dict):
             model = SimpleModel(hidden_dim, nlayers=4)
 
+        params = list(model.parameters())
+        param_groups = [{"params": params[::2], "weight_decay": 0.0}, {"params": params[1::2], "weight_decay": 0.1}]
```

Comment:
I recommend improving readability by making things more explicit like so:

```python
  param_groups = [
      {"params": [p for n, p in model.named_parameters() if not 'bias' in n], "weight_decay" : 0.1}, 
      {"params": [p for n, p in model.named_parameters() if 'bias' in n], "weight_decay" : 0.0}
  ]
```

On a minor note, I also flipped the weight decay to weights instead of biases :).